WinUIGallery/ControlPages/RevealFocusPage.xaml

```diff
@@ -140,7 +140,7 @@
                          />
 
                 <ComboBox MinWidth="300" Margin="0,16,0,0" 
-                          AutomationProperties.Name="Sample ComboBox"
+                          AutomationProperties.Name="Sample"
```

Comment:
shouldn't the names be unique? (lines 143 & 165)

Reply:
In this case as they are different control types, narrator will read: "Sample combobox" and "Sample slider", so users won't be confused.